### ISSUE TITLE
Use workflow_dispatch inputs

### ISSUE THREAD

**Issue by AnimMouse** [AnimMouse/torrent-webseed-creator#12]
Instead of requiring to commit and push the parameters, use instead workflow_dispatch inputs to make it easy.

```
on:
  workflow_dispatch:
    inputs:
      url:
        description: URL
        required: true
        
jobs:
  download-file:
    runs-on: ubuntu-latest
    steps:
      - name: Download file
        run: wget ${{ github.event.inputs.url }}
```

**Comment by AnimMouse** [comment]
Successfully implemented and test @ 70fbce83d12d29c43ffe1ff2d6c062a0a2512be1
Now documentation on how to use is needed.

Also deprecate the old way (editing the environment variables on .yml file) and move to the workflow_dispatch way.

**Comment by AnimMouse** [comment]
Old way deprecated @ fa823e5d36174d47fa12e4d6213a41e15c885b60 and updated instructions on README.md.
